cheddar/cheddar-integration-aws/src/test/java/com/clicktravel/infrastructure/persistence/aws/dynamodb/integration/DynamoDbDataGenerator.java

```diff
@@ -128,6 +130,50 @@ void createStubItemWithRangeTable() throws Exception {
         }
     }
 
+    void createStubItemWithGlobalSecondaryIndexTable() throws Exception {
```

Comment:
I copied/pasted and then modified an existing method in the class.  I have no idea why it was default/package level visibility and can't see any reason why that was the case.  I have corrected this and the many other occurrences of this in the class.